tsconfig.json

```diff
@@ -16,17 +16,7 @@
       "~/*": ["./*"],
```

Comment:
I think we should be able to change the ``target`` and ``module`` options to ESNext just for the tests, that should speed things up as TypeScript won't transpile anything. Worth a try.

Can you also test which of these options are necessary for the global file as well? So it's not a blanket copy of the one in the package.

Reply:
> I think we should be able to change the target and module options to ESNext just for the tests

We should keep these the same per sub-project imo, with the root tsconfig only giving a baseline for config files and such, unless the sub-project doesn't need a special configuration.

I also don't think tests run through Babel, since they run directly in Node.

Reply:
You didn't understand my point I think :)

Basically, what I didn't want is to do unnecessary transpilations for linter/jest/nuxt config files. In Chromecast for instance, we do a transpilation for webpack config and another for the whole client. That's what I wanted to avoid here.

However, I checked out the branch locally and I noticed that there are no TS files in the root folder, so, as you mention, this file is just unused (but I like the idea of having it as a template) :)